tools/gen_addon_readme.py

```diff
@@ -282,8 +292,13 @@ def gen_one_addon_index(readme_filename):
               help="git commit changes to README.rst, if any.")
 @click.option('--gen-html/--no-gen-html', default=True,
               help="Generate index html file.")
+@click.option('--lang', default='',
+              help="Target language file code (default is en_En)"
+                   "List of supported codes: "
+                   "[(--lang=es for es_ES),]")
```

Comment:
I am not sure that covers my point. How would you code MX Spanish?

Reply:
Thought in principle to have just one file per language to not do it more complicated but I understand your concern. So what about now with language + country codes?

Reply:
Indeed that was my intention originally: use standard language+country code @victormmtorres

Reply:
Think it's done, could you update your review?